packages/chat-headless/src/ChatHeadless.ts

```diff
@@ -64,18 +64,7 @@ export class ChatHeadless {
       ...this.config.analyticsConfig,
     });
     if (this.config.saveToSessionStorage) {
```

Comment:
if we want to publish this change now, then you can bump the package version for headless in this PR. Once this is merge and published, we can then make a new commit/PR to update package version(s) for headless-react.